soco/music_library.py

```diff
@@ -139,6 +142,33 @@ def get_playlists(self, *args, **kwargs):
         args = tuple(['playlists'] + list(args))
         return self.get_music_library_information(*args, **kwargs)
 
+    def get_sonos_favorites(self, *args, **kwargs):
+        """Convenience method for `get_music_library_information`
```

Comment:
I know the other docstrings in this class are broken too, but can you fix up the new ones to follow the docstring convention of:
```python
def myfunction():
    """Short preferably one-line description

    Longer description

    Args (not applicable here)
    """
```

it may even have been me that wrote them that way to begin with, anyway, we should get them fixed up.